TourGuide_Horde/29_30_HillsbradFoothills.lua

```diff
@@ -29,7 +29,8 @@ A Elixir of Pain (Part 2) |QID|502|
 T Helcular's Revenge |QID|552|
 A Helcular's Revenge (Part 2) |QID|553|
 
-C Helcular's Revenge (Part 2) |QID|553|
+C Helcular's Revenge (Part 2) |QID|553| |QO|Flame of Azel charged|
+C Helcular's Revenge (Part 2) |QID|553| |QO|Flame of Veraz charged|
```

Comment:
The intent is not to use the |Q| and |QO| tags with C objectives. This is because C completes when the entire quest is complete. Use |Q| and |QO| tags only with N (notes) or K (kill) objectives.

In this particular case, the user understands that both flames need to be charged to complete the quest, and thus just leaving the C objective will suffice. However, if you believe it should be clarified, you can create a note explaining to charge both of these flames and provide a coordinate for each. TourGuide knows when there are multiple coordinates provided in the note and will map both of them in TomTom sequentially.

Reply:
More logic, will change.